src/link/MachO/Object.zig

```diff
@@ -600,6 +600,8 @@ pub fn parseDwarfInfo(self: Object) error{Overflow}!dwarf.DwarfInfo {
                 di.debug_abbrev = try self.getSectionContents(sect);
             } else if (mem.eql(u8, sectname, "__debug_str")) {
                 di.debug_str = try self.getSectionContents(sect);
+            } else if (mem.eql(u8, sectname, "_-debug_str_offsets")) {
```

Comment:
The `_-` prefix looks out of place, may be a typo?

Reply:
It definitely is, thanks @rohlem